examples/Example/src/combo/index.tsx

```diff
@@ -111,18 +134,23 @@ class ControlledSwitch extends React.Component {
   }
 }
 
-class Combo extends Component {
+type ComboPropsType = {
+  ScrollViewComponent: typeof GHScroll | typeof RNScroll;
+};
+
+class Combo extends Component<ComboPropsType> {
   _onClick = () => {
-    alert("I'm so touched");
-    this._scrollView.scrollTo({ y: 200, animated: true });
+    Alert.alert("I'm so touched");
+    this._scrollView?.scrollTo({ y: 200, animated: true });
   };
+  private _scrollView: RNScroll | GHScroll | null = null;
   render() {
     const { ScrollViewComponent } = this.props;
     return (
       <View style={styles.container}>
+        {/* @ts-ignore I cannot find a way to type it properly */}
         <ScrollViewComponent
```

Comment:
Isn't that a problem? Currently, we're passing c-tor for a component into `Combo` and this should have the correct type but it doesn't.

Reply:
I changed it, not an ideal solution, but it works.

Reply:
It is what it is 🤷‍♂️